DESCRIPTION

```diff
@@ -32,35 +32,43 @@ Imports:
     stats,
     tibble (>= 3.1.0),
     tidyr,
-    tune (>= 0.1.3),
+    tune (>= 0.1.6.9000),
     vctrs,
     withr,
-    workflows (>= 0.2.3)
+    workflows (>= 0.2.4.9002)
 Suggests: 
     covr,
-    dials,
+    dials (>= 0.1.0),
     kknn,
     knitr,
     modeldata,
-    parsnip,
-    recipes (>= 0.1.15),
+    parsnip (>= 0.2.0),
+    recipes (>= 0.2.0),
     rmarkdown,
     spelling,
     testthat (>= 3.0.0),
     yardstick
 VignetteBuilder: 
     knitr
 Config/Needs/website: 
+    tidymodels/discrim, 
```

Comment:
I think this one can be switched back to the CRAN version once all the binaries for `0.2.0` are built?

Reply:
yes